sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosContainer.java

```diff
@@ -46,134 +46,124 @@
-     * @return the cosmos sync container response
+     * @param options the options.
+     * @return the Cosmos sync container response.
      */
     public CosmosContainerResponse read(CosmosContainerRequestOptions options) {
         return database.mapContainerResponseAndBlock(this.asyncContainer.read(options));
     }
 
     /**
-     * Delete cosmos sync container response.
+     * Deletes the current cosmos sync container response while specifying additional options such as If-Match.
      *
-     * @param options the options
-     * @return the cosmos sync container response
+     * @param options the options.
+     * @return the cosmos sync container response.
      */
     public CosmosContainerResponse delete(CosmosContainerRequestOptions options) {
         return database.mapContainerResponseAndBlock(this.asyncContainer.delete(options));
     }
 
     /**
-     * Delete cosmos sync container response.
+     * Deletes the current cosmos sync container response.
      *
-     * @return the cosmos sync container response
+     * @return the cosmos sync container response.
      */
     public CosmosContainerResponse delete() {
         return database.mapContainerResponseAndBlock(this.asyncContainer.delete());
     }
 
     /**
-     * Replace cosmos sync container response.
+     * Replaces the current container properties.
      *
-     * @param containerProperties the container properties
-     * @return the cosmos sync container response
+     * @param containerProperties the container properties.
+     * @return the cosmos sync container response.
      */
     public CosmosContainerResponse replace(CosmosContainerProperties containerProperties) {
         return database.mapContainerResponseAndBlock(this.asyncContainer.replace(containerProperties));
     }
 
     /**
-     * Replace cosmos sync container response.
+     * Replaces the current container properties while specifying additional options such as If-Match.
      *
-     * @param containerProperties the container properties
-     * @param options the options
-     * @return the cosmos sync container response
+     * @param containerProperties the container properties.
+     * @param options the options.
+     * @return the cosmos sync container response.
      */
     public CosmosContainerResponse replace(CosmosContainerProperties containerProperties,
                                            CosmosContainerRequestOptions options) {
         return database.mapContainerResponseAndBlock(this.asyncContainer.replace(containerProperties, options));
     }
 
     /**
-     * Read provisioned throughput integer.
+     * Read provisioned throughput for the current container.
      *
-     * @return the integer. null response indicates database doesn't have any provisioned RUs
+     * @return provisioned throughput for the current container or null response if doesn't have any provisioned RUs.
      */
     public Integer readProvisionedThroughput() {
```

Comment:
We should hide / remove this API - as discussed in this ticket : https://github.com/Azure/azure-sdk-for-java/issues/11191

Reply:
Fixed.